rca/landingpages/utils.py

```diff
@@ -0,0 +1,14 @@
+def news_teaser_formatter(item, image=None):
+    item_as_dict = {}
+
+    editorial_type = item.editorial_types.first()
+    item_as_dict["type"] = editorial_type.type if editorial_type else ""
+
+    if image and item.hero_image:
+        item_as_dict["image"] = item.hero_image.file.url
```

Comment:
**Question** I think this will return the URL for the original image, does it need to use a particular rendition?